corehq/form_processor/submission_process_tracker.py

```diff
@@ -63,4 +63,5 @@ def unfinished_archive(instance, user_id, archive):
         )
     tracker = ArchiveProcessTracker(unfinished_archive_stub)
     yield tracker
```

Comment:
if there isn't a stub, does any of this need to happen?

Reply:
https://github.com/dimagi/commcare-hq/pull/22790/commits/4893b1a7248ddd9d7fdc956b8ce06c8855d91ba9
Check out the updated PR description